src/partition/repart.c

```diff
@@ -6094,6 +6103,65 @@ static int parse_private_key(const char *key, size_t key_size, EVP_PKEY **ret) {
 #endif
 }
 
+static int finalize_private_key(EVP_PKEY **ret_private_key, AskPasswordUserInterface **ret_user_interface) {
```

Comment:
Is it not necessary to guard with `#if HAVE_OPENSSL` ?

BTW, naming `finalize` sounds spurious for me. Instead, it looks `initialize`. No?